frontend/src/i18n/messages/en/index.ts

```diff
@@ -572,10 +572,8 @@ export const messages = {
     },
     editModal: {
       loading: "Updating",
-      title: "Edit name of search query",
-      description:
-        "<strong>Tip:</strong> You can’t edit a saved query’s search terms or filters. However, you can apply the query to a new search, make changes, and save it as a new query",
-      inputLabel: "Query name <required>(required)</required>",
+      title: "Edit",
+      inputLabel: "New name",
```

Comment:
it isn't captured in the requirements on this ticket because I think different people were proposing different changes to this modal at different times, but we do still want to mark this as a required field in the input label. Sorry for the confusion!